assets/js/components/Link.js

```diff
@@ -82,6 +87,7 @@ function Link( {
 				},
 			) }
 			href={ isAnchor ? href : undefined }
```

Comment:
Might be better to pass this through only if `href && ! disabled` is true:
```suggestion
			href={ href !== undefined && ! disabled ? href : undefined }
```

Reply:
`isAnchor` was checking for falsy `href` values as well as `! disabled`, but I made the modification to typecheck for undef instead to be more clear. Empty strings aren't a possible value due to the ESLint rule (`jsx-a11y/anchor-is-valid`) which warns against it.